source/extensions/filters/http/istio_stats/istio_stats.cc

```diff
@@ -95,6 +95,11 @@ extractEndpointMetadata(const StreamInfo::StreamInfo& info) {
     const auto& filter_metadata = upstream_host->metadata()->filter_metadata();
     const auto& it = filter_metadata.find("istio");
     if (it != filter_metadata.end()) {
+      std::string identity = "";
```

Comment:
I'm confused - this is using EDS, not WDS?

Reply:
Also an older implementation I forgot to scrub